### ISSUE TITLE
Problem with swift3.0 and XLPagerTabStrinp 6.0.0

### ISSUE THREAD

**Issue by miguelgutierrezhcd** [xmartlabs/XLPagerTabStrip#251]
I have a link error:
I use carthage:
github "xmartlabs/XLPagerTabStrip" == 6.0.0

(If I use carthage: github "xmartlabs/XLPagerTabStrip" == 5.1.0 and swift 2.3 is OK) 

My example: 
class ViewController:  ButtonBarPagerTabStripViewController{
}

Undefined symbols for architecture x86_64:
  "direct field offset for XLPagerTabStrip.PagerTabStripViewController.(containerView.storage in _003637BCF98C7C0D0B48098362580A1B) : __ObjC.UIScrollView!?", referenced from:
      XLPagerTabStrip.PagerTabStripViewController.containerView.setter : __ObjC.UIScrollView! in ViewController.o
      XLPagerTabStrip.PagerTabStripViewController.(containerView.materializeForSet : __ObjC.UIScrollView!).(closure #1) in ViewController.o
  "direct field offset for XLPagerTabStrip.ButtonBarPagerTabStripViewController.(buttonBarView.storage in _0CF27031E34C35EB5EAF50A7E0F72A93) : XLPagerTabStrip.ButtonBarView!?", referenced from:
      XLPagerTabStrip.ButtonBarPagerTabStripViewController.buttonBarView.setter : XLPagerTabStrip.ButtonBarView! in ViewController.o
      XLPagerTabStrip.ButtonBarPagerTabStripViewController.(buttonBarView.materializeForSet : XLPagerTabStrip.ButtonBarView!).(closure #1) in ViewController.o
  "direct field offset for XLPagerTabStrip.ButtonBarPagerTabStripViewController.(buttonBarItemSpec.storage in _0CF27031E34C35EB5EAF50A7E0F72A93) : XLPagerTabStrip.ButtonBarItemSpec<XLPagerTabStrip.ButtonBarViewCell>?", referenced from:
      XLPagerTabStrip.ButtonBarPagerTabStripViewController.buttonBarItemSpec.setter : XLPagerTabStrip.ButtonBarItemSpec<XLPagerTabStrip.ButtonBarViewCell> in ViewController.o
      XLPagerTabStrip.ButtonBarPagerTabStripViewController.(buttonBarItemSpec.materializeForSet : XLPagerTabStrip.ButtonBarItemSpec<XLPagerTabStrip.ButtonBarViewCell>).(closure #1) in ViewController.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by indrajitsinh** [comment]
@miguelgutierrezhcd  same error for me as well , i am using Carthage , but it will working fine using Cocopods , have you found any work around for this ?

**Comment by miguelgutierrezhcd** [comment]
@indrajitsinh No, I haven´t found a solution with Carthage, and I´d like to have it.

**Comment by indrajitsinh** [comment]
@miguelgutierrezhcd  it's the only problem with Carthage , i tried with Cocopods and works fine, so for as of now i just mix Cocopods and Carthage once the issue resolved i delete Cocopods from project

**Comment by dpfbop** [comment]
I have the same issue

**Comment by alengqvist** [comment]
Same issue here, will try with CP instead.

**Comment by jazzsasori** [comment]
I have same issue

**Comment by cocoabagel** [comment]
Same issue

**Comment by marcadam** [comment]
Same issue.

**Comment by Albin-CR** [comment]
I'm having issue with Cocopods also

**Comment by santiagofm** [comment]
Hey @Albin-CR !

You are experiencing the exact same issue with cocoapods? If not could you elaborate a bit more, thanks !

Cheers
